server/service/src/sync/synchroniser.rs

```diff
@@ -304,6 +304,8 @@ impl Synchroniser {
         ctx.processors_trigger
             .trigger_processor(ProcessorType::ContactFormEmail);
 
+        // This should be before plugin processor below, in case there is a processor error, need to be able
+        // to sync new plugin version to avoid bricking the app
```

Comment:
There was a panic, that cause app to quit on startup, after sync, PR description